stacks/_templates/jupyterhub.yaml

```diff
@@ -5,6 +5,8 @@ repo:
   url: https://jupyterhub.github.io/helm-chart/
 version: 3.0.0
 options:
+  labels:
+    stackable.tech/vendor: Stackable
```

Comment:
"Additional property labels is not allowed"
```suggestion
```

Reply:
See https://github.com/stackabletech/demos/pull/9#discussion_r1474638318

Reply:
Error has gone here, too.